apps/studio/schemas/boardYear.ts

```diff
@@ -0,0 +1,67 @@
+import { HeartIcon } from "@sanity/icons";
+
+const organizerSearch = [
```

Comment:
Would `organizers` be a more apt name for this?

Reply:
I feel like it makes more sense for this to be named `organizerSearch` since that's what a user would be doing with these fields.

Reply:
Oh I see what you mean now, I just realized this object only represents a single organizer, not the list.